### ISSUE TITLE
OneJar plugin fails in Gradle 4.6

### ISSUE THREAD

**Issue by nsoft** [nsoft/jesterj#104]
More bit-rot relating to #92

```gus@ns-l1:~/projects/jesterj/code/jesterj/code/ingest$ ./gradlew oneJar

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':oneJar'.
> Cannot convert the provided notation to a File or URI: java.io.BufferedWriter@39d88fd9.
  The following types/formats are supported:
    - A String or CharSequence path, for example 'src/main/java' or '/usr/include'.
    - A String or CharSequence URI, for example 'file:/usr/include'.
    - A File instance.
    - A Path instance.
    - A Directory instance.
    - A RegularFile instance.
    - A URI or URL instance.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
See https://docs.gradle.org/4.6/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 1s
4 actionable tasks: 2 executed, 2 up-to-date
```

**Comment by nsoft** [comment]
https://github.com/rholder/gradle-one-jar/issues/34

**Comment by nsoft** [comment]
Fixed in #129
